pkg/controller/nexus/resource/route.go

```diff
@@ -33,7 +34,6 @@ const (
 type routeBuilder struct {
 	route *v1.Route
 	err   error
-	nexus *v1alpha1.Nexus
```

Comment:
Nice! This was one of the things I mentioned I wanted to change, completely unnecessary, but I missed it the first time hahaha :tada: :grin:

Reply:
Lint remembered. hahaha